.github/scripts/buildActions.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Used to precompile all Github Action node.js scripts using ncc.
+# This bundles them with their dependencies into a single executable node.js script.
+
+# In order for this script to be safely run from anywhere, we cannot use the raw relative path '../actions'
+declare -r ACTIONS_DIR="$(dirname "$(dirname "$(realpath "$0")")")/actions"
+
+# List of paths to all JS files that implement our GH Actions
+declare -r GITHUB_ACTIONS=(
+    "$ACTIONS_DIR/bumpVersion/bumpVersion.js"
+)
+
+# This will be inserted at the top of all compiled files as a warning to devs.
+declare -r NOTE_DONT_EDIT='/**
+ * NOTE: This is a compiled file. DO NOT directly edit this file.
```

Comment:
While this is nice, it feels like some over engineering. If the user edits the prebuilt file, will the action fail the PR and they will realize that way? I just wonder if we can 100% this script and just change `gh-actions-build` back to `ncc build ...`?

Reply:
Sorry, I'm a bit unsure of what you're suggesting here. This script was created when I had a different idea of how the verification script would work. However, I left it because:

1) It works
2) It makes it easier to add new actions - just add the path to the `GITHUB_ACTIONS` variable

The note was just a bonus, but I could take it out if you feel it's adding undesirable complexity.

Reply:
I was trying to think if there was a way to replace this entire script with a bash one liner we can put into `package.json` instead, but I can't think of one that would replace this 100%. The fact that we need to build each action into it's own `index.js` is one reason to keep the script.

Reply:
Ahh, gotcha. I think we _could_ do something like:

```bash
ncc build .github/actions/bumpVersion/bumpVersion.js -o .github/actions/bumpVersion \
    && ncc build .github/actions/someOtherAction/someOtherAction.js -o .github/actions/someOtherAction \
    && ....
    && ....
```

But I think the script is cleaner. In hindsight I might not have written it because it's not _that_ necessary, but now that it's been done I think we're better off keeping it.